frontend/public/components/storage/create-pvc.tsx

```diff
@@ -269,6 +269,7 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
             key={radio.value}
             onChange={handleVolumeMode}
             inline
+            disabled={storageProvisioner.includes('cephfs') && radio.value === 'Block'}
```

Comment:
Declare const in `frontend/public/components/storage/shared.ts` file for `CephFS` like we have it for RBD and then use it.